storage-transfer/src/main/java/com/google/cloud/storage/storagetransfer/samples/QuickstartSample.java

```diff
@@ -0,0 +1,40 @@
+package com.google.cloud.storage.storagetransfer.samples;
+
+import com.google.storagetransfer.v1.proto.StorageTransferServiceClient;
+import com.google.storagetransfer.v1.proto.TransferProto.CreateTransferJobRequest;
+import com.google.storagetransfer.v1.proto.TransferTypes.GcsData;
+import com.google.storagetransfer.v1.proto.TransferTypes.TransferJob;
+import com.google.storagetransfer.v1.proto.TransferTypes.TransferSpec;
+
+public class QuickstartSample {
+  /** Quickstart sample using transfer service to transfer from one GCS bucket to another. */
```

Comment:
Hey, just now seeing this - I think either is fine, but `runTransferJob` might be a bit more complex for the customer (a user would have to know to disable to job). While more verbose, I think scheduling would make more sense (although it is a somewhat of a misnomer if we want to run this one time). For this reason, I think a one-time helper function would be nice to have across the GAPIC clients.

Reply:
What do you mean by "a user would have to know to disable to job"?

Jobs that are created and then run this way, only run once. There's no need to disable it

Reply:
You're correct that there isn't a reason to explicitly disable it; I'm thinking someone new to the product may be concerned because 'Frequency' isn't set and the documentation mentions ['Schedule one-time transfer operations or recurring transfer operations'](https://cloud.google.com/storage-transfer/docs/overview#what-is-storage-transfer), making it a bit ambiguous if this transfer job will run again in the future.

Setting the schedule to the same date sets the Frequency to 'Once' in the UI - making it clearer to the user that this won't run again in the future.

<img width="1665" alt="Screen Shot 2021-11-12 at 8 04 47 AM" src="https://user-images.githubusercontent.com/10792787/141497641-f0f13719-22e0-462d-9a0c-ba76c338058f.png">

I think this would be valuable to capture as feedback and share upstream